homeassistant/components/plugwise/quality_scale.yaml

```diff
@@ -0,0 +1,89 @@
+rules:
+  ## Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry:
+    status: todo
+    comment: Add tests preventing second entry for same device
+  config-flow-test-coverage:
+    status: todo
+    comment: Cover test_form and zeroconf
+  runtime-data:
+    status: todo
+    comment: Clean up test_init for testing internals
+  test-before-setup: done
+  appropriate-polling:
+    status: todo
+    comment: Clean up coordinator (L71) check for mypy happiness
+  entity-unique-id: done
+  has-entity-name:
+    status: todo
+    comment: Clean up climate (already in superclass)
+  entity-event-setup: done
+  dependency-transparency: done
+  action-setup:
+    status: todo
+    comment: Check if we have these, otherwise exempt
+  common-modules:
+    status: todo
+    comment: Verify entity for async_added_to_hass usage (discard?)
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  docs-actions: done
+  brands: done
+  ## Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions:
+    status: todo
+    comment: Climate exception on ValueError should be ServiceValidationError
+  reauthentication-flow:
+    status: exempt
+    comment: The hubs have a hardcoded `Smile ID` printed on the sticker used as password, it can not be changed
+  parallel-updates:
+    status: todo
+    comment: Using coordinator, but required due to mutable platform
+  test-coverage:
+    status: todo
+    comment: Consider using snapshots + consistency in setup calls + add numerical tests + use fixtures
+  integration-owner: done
+  docs-installation-parameters: done
+  docs-configuration-parameters: done
+  ## Gold
+  entity-translations:
+    status: todo
+    comment: Clean up name where not needed, remove translation key on deviceclasses
+  entity-device-class: done
+  devices: done
+  entity-category: done
+  entity-disabled-by-default: done
+  discovery: done
+  stale-devices: done
+  diagnostics: done
+  exception-translations:
+    status: todo
+    comment: Add coordinator, util and climate exceptions
+  icon-translations: done
+  reconfiguration-flow:
+    status: todo
+    comment: This integration does not have any reconfiguration steps (yet) investigate how/why
+  dynamic-devices:
+    status: todo
+    comment: Add missing logic to button for unloading and creation
+  discovery-update-info: done
+  repair-issues:
+    status: exempt
+    comment: This integration does not have repairs
+  docs-use-cases: done
+  docs-supported-devices: done
+  docs-supported-functions: done
+  docs-data-update: done
+  docs-known-limitations: done
+  docs-troubleshooting: done
+  docs-examples: done
```

Comment:
I am missing a few of these in the docs

Reply:
All set to todo just for re-checking one by one